src/System.Net.HttpListener/tests/HttpResponseStreamTests.cs

```diff
@@ -478,8 +478,7 @@ public void Dispose()
 
                 // Disconnect the Socket from the HttpListener.
                 client.Close();
-                GC.Collect();
-                GC.WaitForPendingFinalizers();
+                Thread.Sleep(2000);
```

Comment:
(If you wanted to remove this commit, I think the rest of the change is good to go in.)